### ISSUE TITLE
Broken sudoers configuration when using jkirk.users multiple times

### ISSUE THREAD

**Issue by mika** [jkirk/ansible-role-user#4]
Given the following setup:

```
synpromika@pve01 ~ % sudo cat /etc/sudoers.d/adm 
# HEADER: managed by ansible, do NOT edit manually!
foo ALL=NOPASSWD: ALL
synpromika ALL=NOPASSWD: ALL
```

Then updating the ansible user playbook to add also `synprodpat` user, but use `bash` for existing user `foo` and `zsh` for `synpromika` + `synprodat` via:

```
- name: Setup of users
  hosts: pve01.example.com
  become: yes
  roles:
    - { role: jkirk.user, users: [ 'foo'], groupname: 'adm', admin: True, user_shell: '/bin/bash' }
    - { role: jkirk.user, users: [ 'synpromika', 'synprodpat'], groupname: 'adm', admin: True, user_shell: '/usr/bin/zsh' } 
```

This then leads to the following result:

```
TASK [jkirk.user : Install sudoers file for group adm] *************************
--- before: /etc/sudoers.d/adm
+++ after: /home/mika/.ansible/tmp/ansible-local-173248khd2eovb/tmp2dfzuy9z/admin.j2
@@ -1,3 +1,2 @@
 # HEADER: managed by ansible, do NOT edit manually!
 foo ALL=NOPASSWD: ALL
-synpromika ALL=NOPASSWD: ALL

changed: [pve01.example.com]
[...]
TASK [jkirk.user : Install sudoers file for group adm] *************************
--- before: /etc/sudoers.d/adm
+++ after: /home/mika/.ansible/tmp/ansible-local-173248khd2eovb/tmpui00pgem/admin.j2
@@ -1,3 +1,3 @@
 # HEADER: managed by ansible, do NOT edit manually!
-foo ALL=NOPASSWD: ALL
 synpromika ALL=NOPASSWD: ALL
+synprodpat ALL=NOPASSWD: ALL
```

So it first seems to remove the existing configuration for `synpromika`, then later on removes the `foo` user. 

AFAICT it's not possible to use the `jkirk.user` role multiple times due to the way the templates file is used and iteration over it is handled?

**Comment by jkirk** [comment]
Yes, this is by  "design": This role is meant to bootstrap / deploy our sysadmin user accounts with SSH public key based logins and passwordless sudo.

All sysadmin users are meant to have the same setup (if they belong to the same primary group (aka `groupname`).
`groupname` shouldn't be an existing group (although it works), instead use something like "sysadmin" or "synpro".

For additional groups (which may or may not exist) use `user_groups`.

For something more sophisticated use a different role, the ansible-galaxy role [robertdebock.users](https://galaxy.ansible.com/robertdebock/users) has proven useful.

I consider this issue as wontfix. If you disagree, re-open this issue and add it our meeting agenda.
